Docker.DotNet/Models/HostConfig.cs

```diff
@@ -12,6 +12,9 @@ public class HostConfig
         [DataMember(Name = "Links")]
         public IList<string> Links { get; set; }
 
+        [DataMember(Name = "GroupAdd")]
+        public IList<string> GroupAdd { get; set; }
```

Comment:
Yeah, I found it unusual that it wasn't listed in the docs themselves and only in the api changes. I had to check the source code to confirm its existence and to determine what type it was.